src/benchmarks/micro/libraries/System.Collections/PriorityQueue/Perf_PriorityQueue.cs

```diff
@@ -106,5 +106,30 @@ public void K_Max_Elements()
                 queue.Dequeue();
             }
         }
+
+        [Benchmark]
+        public void KMaxPriorityQueueSize_DequeueEnqueue_PerformanceRegressionBenchmark()
```

Comment:
IIRC this isn't really computing any max or min elements, since unlike `EnqueueDequeue`, `DequeueEnqueue` is never comparing the extracted min heap entry with the inserted entry. The `PerformanceRegressionBenchmark` suffix is redundant, since that's what all benchmarks in this repo are intended for. Recommend keeping the benchmark name ambiguous:

```suggestion
        public void DequeueEnqueue()
```